packages/blade/src/_helpers/storybook/ComponentStatusTable.tsx

```diff
@@ -0,0 +1,241 @@
+  },
+  {
+    name: 'TextInput',
+    status: 'released',
+    releasedIn: '2.1.0',
+  },
+  {
+    name: 'Title',
+    status: 'released',
+    releasedIn: '0.5.0',
+  },
+  {
+    name: 'VisuallyHidden',
+    status: 'released',
+    releasedIn: '0.9.0',
+  },
+];
+
+const Table = styled.table`
+  width: 100%;
+  border-collapse: collapse;
+  table-layout: fixed;
+
+  th,
+  td {
+    padding: ${({ theme }) => makeSpace(theme.spacing[4])};
+    padding-inline: ${({ theme }) => makeSpace(theme.spacing[7])};
+  }
+
+  tr:nth-child(odd) {
+    background-color: ${({ theme }) => theme.colors.brand.gray[200].lowContrast};
+  }
+
+  tr:nth-child(even) {
+    background-color: ${({ theme }) => theme.colors.brand.gray[300].lowContrast};
+  }
+`;
+
+const ComponentStatusBadge = ({ status }: { status: ComponentStatuses }): React.ReactElement => {
+  const badgeVariants: Record<
+    ComponentStatuses,
+    { label: string; variant: BadgeProps['variant']; icon: IconComponent }
+  > = {
+    released: { label: 'Released', variant: 'positive', icon: CheckIcon },
+    deprecated: { label: 'Deprecated', variant: 'negative', icon: AlertCircleIcon },
+    'in-design': { label: 'In Design', variant: 'notice', icon: LoaderIcon },
+    'in-api-spec': { label: 'API In Progress', variant: 'notice', icon: EditIcon },
+    'in-development': { label: 'In Development', variant: 'notice', icon: LoaderIcon },
+    'planned-Q1': { label: 'Planned: Q1', variant: 'information', icon: ClockIcon },
+    'planned-Q2': { label: 'Planned: Q2', variant: 'information', icon: ClockIcon },
+    'planned-Q3': { label: 'Planned: Q3', variant: 'information', icon: ClockIcon },
+    'planned-Q4': { label: 'Planned: Q4', variant: 'information', icon: ClockIcon },
+  };
+
+  return (
+    <Badge variant={badgeVariants[status].variant} icon={badgeVariants[status].icon}>
+      {badgeVariants[status].label}
+    </Badge>
+  );
+};
+
+const ReleasedInLink = ({ version }: { version?: string }): React.ReactElement => {
+  const ghUrl = 'https://github.com/razorpay/blade/releases/tag/%40razorpay%2Fblade%40';
+  return version ? <Link href={`${ghUrl}${version}`}>{`v${version}`}</Link> : <Text>-</Text>;
+};
+
+const ComponentStatusTable = (): React.ReactElement => {
+  const sortedComponentData = componentData.sort((prev, next) => {
+    return prev.name.localeCompare(next.name);
+  });
+
+  return (
+    <Box>
+      <Table>
+        <thead>
+          <tr>
+            <th align="left">
+              <Text weight="bold">Component</Text>
+            </th>
+            <th align="left">
+              <Text weight="bold">Status</Text>
+            </th>
+            <th align="right">
+              <Text weight="bold">Released In</Text>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {sortedComponentData.map((data) => {
+            return (
+              <tr key={data.name}>
+                <td align="left">
+                  <Text>{data.name}</Text>
```

Comment:
Use link here and put a href to components story link

Reply:
Done. phew! It was quite tricky to get the storybook api to do redirection.